editions/test/tiddlers/tests/data/let-filter-prefix/ResultList.tid

```diff
@@ -5,7 +5,7 @@ tags: [[$:/tags/wiki-test-spec]]
 
 title: Output
 
-<$text text={{{ [all[tiddlers]] :let[[varname]] [varlist[varname]sort[]join[,]] }}}/>
+<$text text={{{ [all[tiddlers]] :let[[varname]] [(varname)sort[]join[,]] }}}/>
```

Comment:
I do like this new syntax. IMO it looks very clean.

Reply:
It's a pleasurable surprise that we've been able to simplify things to this extent. I've got one more update I'm working on now as well.